lib/iris/tests/unit/fileformats/grib/save_rules/test_product_definition_template_40.py

```diff
@@ -52,10 +53,10 @@ def test_constituent_type(self, mock_set):
 
         product_definition_template_40(cube, mock.sentinel.grib)
         mock_set.assert_any_call(mock.sentinel.grib,
-                                 "productDefinitionTemplateNumber", 40)
+                                 'productDefinitionTemplateNumber', 40)
         mock_set.assert_any_call(mock.sentinel.grib,
-                                 "constituentType", 0)
+                                 'constituentType', 0)
 
 
-if __name__ == "__main__":
+if __name__ == '__main__':
```

Comment:
I have notice that some test uses single quotes(`'string'`) whereas others use double quotes (`"string"`).

Of course this shouldn't be dealt with in this PR, but it's interesting this file is specifically changed from double quotes to single